pom.xml

```diff
@@ -21,7 +21,7 @@
 
     <properties>
         <version.nexus.index>5</version.nexus.index>
-        <version.windup>2.5.0.CR3</version.windup>
+        <version.windup>${project.version}</version.windup>
```

Comment:
again, better to use 2.5.0-SNAPSHOT instead of project.version see http://maven.apache.org/guides/introduction/introduction-to-the-pom.html#Project_Interpolation